### PR TITLE
PR: Employee Admission Form Changed

### PR DESCRIPTION
**Pull request by VijayT8**
ADDED: Employee Admission Form: Updated the final.html

### CODE REVIEW

File: Vijay/Employee Admission Form/employee.html [hunk 1]
```diff
@@ -103,7 +103,6 @@ <h2>WELCOME TO THE REGISTERATION PAGE</h2>
             var Qualification = document.getElementById("qua").value; 
             var Department = document.getElementById("dept").value;
             
-            
             let text = {Resume:Resume,Fname:Fname,Lname:Lname,DOB:DOB,Contact:Contact,Address:Address,
```
Comment (NavinAT) [inline Vijay/Employee Admission Form/employee.html:106]: Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.

File: Vijay/Employee Admission Form/final.html [hunk 2]
```diff
@@ -47,21 +46,6 @@
...
-            td = obj[i].getElementsByTagName("td");
-            if (td)
-              {
-                obj[i].style.display = "";
-              }
-            }       
-          }
-        
 
         function backbut (){
```
Comment (NavinAT) [inline Vijay/Employee Admission Form/final.html:50]: Move the code to separate JS file and indent the code properly

**Review by NavinAT — COMMENTED** [review]
Reviewed your code please do the changes

File: Vijay/Employee Admission Form/employee.html (hunk 1 above)
Comment (VijayT8) [reply Vijay/Employee Admission Form/employee.html:106]: Completed the given Reviews bro.

File: Vijay/Employee Admission Form/final.html (hunk 2 above)
Comment (VijayT8) [reply Vijay/Employee Admission Form/final.html:50]: Completed the given Reviews bro.

File: Vijay/Employee Admission Form/final.html [hunk 3]
```diff
@@ -47,21 +46,6 @@
         var obj = JSON.parse(save);
         
         
-        
-        function myFunction() {
```
[the remark this answers is not in the document]
Comment (VijayT8) [reply Vijay/Employee Admission Form/final.html:51]: Completed the given Reviews bro.

**Pull request merged**